### PR TITLE
MONGOSH-304 - Support server stats and info methods

### PR DESCRIPTION
**Pull request by aherlihy**

### CODE REVIEW

File: packages/cli-repl/src/format-output.ts [hunk 1]
```diff
@@ -71,6 +75,12 @@ function formatDatabases(output): string {
   return textTable(tableEntries, { align: ['l', 'r'] });
 }
 
+function formatStats(output): string {
```
Comment (lrlna) [inline packages/cli-repl/src/format-output.ts:78]: I don't think there is a test for this result type yet.

File: packages/browser-repl/src/components/shell-output-line.tsx [hunk 2]
```diff
@@ -56,6 +56,13 @@ export class ShellOutputLine extends Component<ShellOutputLineProps> {
       return <ShowDbsOutput value={value} />;
     }
 
+    if (type === 'StatsResult') {
```
Comment (lrlna) [inline packages/browser-repl/src/components/shell-output-line.tsx:59]: I think this also needs a test.

File: packages/cli-repl/src/format-output.ts [hunk 3]
```diff
@@ -71,6 +75,12 @@ function formatDatabases(output): string {
...
+  return Object.keys(output).reduce((str, c) => {
+    return `${str}\n${c}\n${inspect(output[c])}\n---\n`;
```
Comment (lrlna) [inline packages/cli-repl/src/format-output.ts:80]: up to you if you want this suggestion:
```suggestion
    return `${str}\n${clr(c, ['bold', 'yellow'])}\n\n${inspect(output[c])}\n---\n`;
```

Comment (aherlihy) [reply packages/cli-repl/src/format-output.ts:80]: Oh woah I didn't realize I could just press 'commit suggestion', that's a sick feature! I just added it by hand.

**Pull request merged**